packages/web-runtime/src/App.vue

```diff
@@ -50,7 +50,11 @@
           v-else
           ref="modalComponent"
           :modal="modal"
-          v-bind="modal.customComponentAttrs"
+          v-bind="
+            typeof modal.customComponentAttrs === 'function'
+              ? modal.customComponentAttrs()
+              : modal.customComponentAttrs
+          "
```

Comment:
We will have this logic in a lot of places in the future I assume - maybe worth adding a helper function for this?

Reply:
Hate it (the `if function then call else use value`), that was exactly the reason why I went for `always the callback` when I adjusted the `SideBarPanel` interface...

Helper would be ok-ish, but the helper might also advocates such type definitions. I'm also fine without the helper. 🤷

Reply:
Then let's make it always a function, I need to rebase anyway.